spotify-artist-viewer/SAArtistViewController.m

```diff
@@ -0,0 +1,110 @@
+//
+//  SAArtistViewController.m
+//  spotify-artist-viewer
+//
+//  Created by Randall Spence on 6/9/15.
+//  Copyright (c) 2015 Intrepid. All rights reserved.
+//
+
+#import "SAArtistViewController.h"
+#import "SAArtist.h"
+#import "SASearchViewController.h"
+#import <QuartzCore/QuartzCore.h>
+@interface SAArtistViewController () <UIScrollViewDelegate>
+@property (strong, nonatomic) IBOutlet UIScrollView *scrollView;
+
+@end
+
+@implementation SAArtistViewController
+
+static NSString * const SAArtistsSearchUrl = @"http://developer.echonest.com/api/v4/artist/biographies?api_key=KUAKSGNM0ERIJ3ZO8&id=spotify:artist:%@";
+
+-(void)viewWillAppear:(BOOL)animated{
+    [super viewWillAppear:animated];
+    
+    self.artistNameLabel.text = self.artist.name;
+    
+    NSData *imageData = [NSData dataWithContentsOfURL:[NSURL URLWithString:self.artist.imageURL]];
+    
+    self.artistImageView.image = [UIImage imageWithData:imageData];
+    
+    self.artistImageView.layer.cornerRadius = self.artistImageView.frame.size.height / 2;
```

Comment:
Use CGRectGetHeight(frame) instead of frame.size.height.

See style guide: https://github.com/IntrepidPursuits/objective-c-style-guide#cgrect-functions